src/realm/table.cpp

```diff
@@ -1687,7 +1687,7 @@ bool Table::has_search_index(size_t col_ndx) const noexcept
 }
 
 
-void Table::upgrade_file_format(size_t target_file_format_version)
+void Table::rebuild_search_index(size_t current_file_format_version)
```

Comment:
I think we should have a function for each kind of upgrade. Like we have now `upgrade_olddatetime` and `rebuild_search_index`. If we should keep the old name, it should do all kinds of upgrades.